### PR TITLE
htop: refresh and add more information link

### PR DESCRIPTION
**Pull request by CleanMachine1**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

Applicable to #6062

### CODE REVIEW

File: pages/common/htop.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
...
 `htop`
 
-- Start htop displaying only processes owned by given user:
+- Start htop displaying processes owned by a specific user:
 
-`htop -u {{username}}`
+`htop --username {{username}}`
 
-- Sort processes by a column (use `--sort-key help` for a column list):
+- Sort processes with a specified parameter (use `htop --sort help` for available parameters):
```
Comment (bl-ue) [inline pages/common/htop.md:14]:
```suggestion
- Sort processes by a specified parameter (use `htop --sort help` for available parameters):
```

File: pages/common/htop.md [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 
-`htop -s {{column_name}}`
+`htop --sort {{parameter}}`
 
 - Get help about interactive commands:
```
Comment (bl-ue) [inline pages/common/htop.md:18]:
```suggestion
- Display help about interactive commands:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by bl-ue — APPROVED** [review]

**Comment by CleanMachine1** [comment]
I think I need to revise this page, I don't like it still

Help desc is too long

And I don't like parameter although it is correct, maybe sort_item?

File: pages/common/htop.md [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
+- Sort processes by a specified `sort_item` (use `htop --sort help` for available options):
 
-`htop -s {{column_name}}`
+`htop --sort {{sort_item}}`
 
-- Get help about interactive commands:
+- Display help:
 
-`?`
+`htop --help`
```
Comment (navarroaxel) [inline pages/common/htop.md:20]: It's ok if you want to add the `--help` example, but the `?` for the interactive mode is really helpful in this page.

Comment (bl-ue) [reply pages/common/htop.md:20]: @CleanMachine1 how about adding both examples?

Comment (CleanMachine1) [reply pages/common/htop.md:20]: hmm, I really wondered what it meant by ?

I thought the original creator had just put, "I don't know how to get help"

Maybe change the desc to something like 

```
- Whilst running htop, see interactive commands:
```

Comment (sbrl) [reply pages/common/htop.md:20]: No, in this case it's a key that you press when htop is actually running, which then displays the inbuilt help on how to use it. That description sounds ok though @CleanMachine1.

Comment (CleanMachine1) [reply pages/common/htop.md:20]: I'll add it now.

Comment (CleanMachine1) [reply pages/common/htop.md:24]: https://github.com/tldr-pages/tldr/pull/6165/commits/1af5616de26967028c09a19ccd4ef0ccf71bffb2

**Review by nicokosi — APPROVED** [review]
👍

**Review by marchersimon — APPROVED** [review]

**Review by sbrl — APPROVED** [review]

**Pull request merged**